doc/ref_model/chapters/chapter08.md

```diff
@@ -98,8 +98,63 @@ The Cloud Services and the Cloud Resources Brokers provide value-added services
 
 <a name="8.2.4"></a>
 ### 8.2.4 Multi-Cloud Interactions Model
-<a name="8.2.4"></a>
-
+<a name="8.2.4.1"></a>
+### 8.2.4.1 Introduction
+To realise a federated cloud requires the definition and agreement on a set of APIs. These APIs should allow each of the parties to interact cooperatively and need to cover the management layer: business management and service operations interactions; as well as the data plane, customer and user, transactions and conversational interfaces.
+
+As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Resource Broker". The set of interactions that these interface points need to provide are defined by the "Multi-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-02.png" alt="Multi-Cloud Interactions Model"></p>
+<p align="center"><b>Figure 8-4:</b> Multi-Cloud Interactions Model</p>
+
+The model defines the following core roles:
+- Communications Service Provider (CSP) - is the party responsible for providing end user service to their customer
+- Customer / User - are the parties that use the service (User) and establishes the business agreement for the service provision (Customer). For retail services the customer and user are the same party, while for enterprise services the Enterprise is the Customer (responsible for the business agreement) and its representatives are the Users.
+- Cloud Providers - are the parties providing the cloud services. These services could be any XaaS service. It could be that a CSP has an agreement with a SaaS Cloud, which in turn uses an IaaS Cloud Provider to deliver their service.
```

Comment:
How match do we have SaaS in the scope of Anuket. I'm not aware of any production ready SaaS solution.

Reply:
@CsatariGergely @walterkozlowski

your question: "How much do we have SaaS in the scope of Anuket?" - I think this should be discussion for next meeting:

For Cloud Provider perspective it is important that SaaS Providers needs are supported.

For the "Multi-Cloud Interactions Model" as looking at general "XaaS" cases and in looking at "Brokerage" across Cloud Providers it is important that SaaS Cloud Provider consumption is considered to ensure Interactions have broad coverage and can be "paper verified" to cover broad range of scenarios.

On your point: "I'm not aware of any production ready SaaS solution" , could you please elaborate, as there is not question that there are lots of existing "production ready SaaS solution" I believe that you are making this point in a much more constrained and specific context.

Thank you for feedback.

Reply:
@CsatariGergely Figure 8-3 covers SaaS, PaaS and IaaS. As you know in the hyperscalar world there is no pure IaaS. And as we start to create complex services, we may see them as consisting of NFs that are on IaaS/PaaS and a set of workloads (and tools) that are SaaS.